### PR TITLE
Fixing broken links, grammatical and punctuation errors in accordproject's documentation

### PR DESCRIPTION
**Pull request by prakharsr**
In reference to my issue at https://github.com/accordproject/techdocs/issues/336

### CODE REVIEW

**Review by Michael-Grover — APPROVED** [review]
I checked the links and grammar in the Netlify preview, it looks good to me, thanks! I'm not sure if we need the approval of someone more familiar with the docs though?

File: docs/logic-simple-expr.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Conditional expressions can be used to perform different computations depending
 
 > Typing `return if 1.0 < 0.0 then "negative" else "positive"` in the [Ergo REPL](https://ergorepl.netlify.com), should answer `Response. "positive" : String`.
 
-See also the [Conditional Expression Reference](ergo-reference.html#condition-expressions)
+See also the [Conditional Expression Reference](ref-logic.html#condition-expressions)
```
Comment (irmerk) [inline docs/logic-simple-expr.md:63]: This should be `ref-ergo-spec.html#condition-expressions`

File: docs/tutorial-nodejs.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ title: With Node.js
 
 You can work with Accord Project templates directly in JavaScript using Node.js.
 
-Documentation for the API can be found in [Cicero API](https://docs.accordproject.org/docs/cicero-api.html).
+Documentation for the API can be found in [Cicero API](ref-cicero-api).
```
Comment (irmerk) [inline docs/tutorial-nodejs.md:10]: This should be `ref-cicero-api.html`

**Review by irmerk — CHANGES_REQUESTED** [review]
Nice! Thanks for all the fixes! A few changes listed below.

Also, anything under `website/versioned_docs/*` should be left alone, those are files generated automatically in the past.

File: docs/logic-simple-expr.md (hunk 1 above)
Comment (prakharsr) [reply docs/logic-simple-expr.md]: Fixed in 9221b7ce9327dad1b1e97cb99ed3608b44baa22c

File: docs/tutorial-nodejs.md (hunk 2 above)
Comment (prakharsr) [reply docs/tutorial-nodejs.md]: Fixed in 9221b7ce9327dad1b1e97cb99ed3608b44baa22c

**Comment by prakharsr** [comment]
@irmerk I applied the required changes to versioned docs in c3b088c8cc8908f3149a2b45d8892027a74aff93 but I found that some of the links were still broken. I read upon the Docusaurus v1 documentation https://docusaurus.io/docs/en/versioning and it is mentioned that individual changes have to be applied to the old versioned docs if they are to be changed. Also, the reverting of versioned docs breaks some of the links for the current stable version 0.21 as some its files differ from the latest version in ```website/docs/```. The next version of documentation that is labelled unstable and is the latest would then be having all the correct links that were applied in the previous commits. I can revert the changes targeted in commit c3b088c8cc8908f3149a2b45d8892027a74aff93 and it would fix the broken links for each version of the documentation.

**Comment by irmerk** [comment]
Hmm, @jeromesimeon do you mind weighing in on this, I'm not as familiar with Docusaurus.

**Comment by prakharsr** [comment]
> @irmerk I applied the required changes to versioned docs in [c3b088c](https://github.com/accordproject/techdocs/commit/c3b088c8cc8908f3149a2b45d8892027a74aff93) but I found that some of the links were still broken. I read upon the Docusaurus v1 documentation https://docusaurus.io/docs/en/versioning and it is mentioned that individual changes have to be applied to the old versioned docs if they are to be changed. Also, the reverting of versioned docs breaks some of the links for the current stable version 0.21 as some its files differ from the latest version in `website/docs/`. The next version of documentation that is labelled unstable and is the latest would then be having all the correct links that were applied in the previous commits. I can revert the changes targeted in commit [c3b088c](https://github.com/accordproject/techdocs/commit/c3b088c8cc8908f3149a2b45d8892027a74aff93) and it would fix the broken links for each version of the documentation.

Hey @mttrbrts, can you please look into the above issue.

**Comment by dselman** [comment]
> @irmerk I applied the required changes to versioned docs in [c3b088c](https://github.com/accordproject/techdocs/commit/c3b088c8cc8908f3149a2b45d8892027a74aff93) but I found that some of the links were still broken. I read upon the Docusaurus v1 documentation https://docusaurus.io/docs/en/versioning and it is mentioned that individual changes have to be applied to the old versioned docs if they are to be changed. Also, the reverting of versioned docs breaks some of the links for the current stable version 0.21 as some its files differ from the latest version in `website/docs/`. The next version of documentation that is labelled unstable and is the latest would then be having all the correct links that were applied in the previous commits. I can revert the changes targeted in commit [c3b088c](https://github.com/accordproject/techdocs/commit/c3b088c8cc8908f3149a2b45d8892027a74aff93) and it would fix the broken links for each version of the documentation.
> Anyone else @jeromesimeon @DianaLease @dselman @mttrbrts familiar with Docusaurus ?

Let's not bother going back and fixing links in old version - just fix the current version.

**Review by dselman — APPROVED** [review]
LGTM

**Pull request merged**